### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by joshgillies** [wd42/wd42-wagtail#2]
This is something I've been meaning to get onto for a while now. But better late than never I guess.

I propose we adopt similar to: https://github.com/confcodeofconduct/confcodeofconduct.com

Thoughts?

**Comment by joshgillies** [comment]
Ideally we'll have this live by the next meetup. As such I'll add a due date to this issue with the hopes to have it online before the next announcement goes out.

**Comment by jonnyscholes** [comment]
Yep this sounds rad. I'm hoping to find some time to work on each event having an end point in preparation for the videos so I'll take this into account when working on that.
